### ISSUE TITLE
Validation still doesn't work...

### ISSUE THREAD

**Issue by noahlocke** [epartment/nova-dependency-container#84]
I'm using the `HasDependencies` trait on my resource.

```php
            BelongsTo::make('County')

            NovaDependencyContainer::make([
                Text::make('Supervisor')->rules('required')
            ])->dependsOn('county.id', 1)
```

**Comment by wize-wiz** [comment]
@noahlocke does `county.id` belongs to a `BelongsTo` field?

**Comment by noahlocke** [comment]
@wize-wiz yes

**Comment by noahlocke** [comment]
@wize-wiz I think you fixed this in c69b2a3
